locale/fi/LC_MESSAGES/django.po

```diff
@@ -3485,3 +3485,74 @@ msgstr "Unohda automaattinen uudelleenohjaus."
 #: userprofile/views.py:80
 msgid "No privacy notice. Please notify administration!"
 msgstr "Ei tietosuojailmoitusta. Ota yhteys ylläpitäjään!"
+
+
+#: results.html
```

Comment:
Use `./manage.py makemessages -i 'venv/*'` to update this file.

Reply:
Done. However, there are some translations that are basically the same in `django.po` and `teacher.fi.json`. Would there be a clean way to use `_()` notation inside the html files?

Reply:
One day, teacher.fi.json will be generated from django.po or so. Untill then, we need to copy translations two two files.